addons/crm/tests/test_crm_lead.py

```diff
@@ -689,3 +689,21 @@ def test_no_date_closed_update_between_won_stages(self):
             leads.stage_id = self.stage_team1_won
         self.assertEqual(won_lead.date_closed, date_closed)
         self.assertEqual(self.lead_1.date_closed, datetime.strptime('2020-02-02 18:00', '%Y-%m-%d %H:%M'))
+
+    @users('admin')
+    def test_crm_lead_company_currency(self):
+        company = self.env['res.company'].create({
```

Comment:
You can use self.company_main instead, available in test sales common. Just force its currency in setup or directly here in the test